packages/js/components/src/experimental-select-tree-control/stories/index.tsx

```diff
@@ -95,6 +96,72 @@ export const MultipleSelectTree: React.FC = () => {
 	);
 };
 
+export const SingleWithinModalUsingBodyDropdownPlacement: React.FC = () => {
+	const [ isOpen, setOpen ] = useState( true );
+	const [ value, setValue ] = React.useState( '' );
```

Comment:
Is there a reason we're calling `useState` as a property instead of directly? Nit picky, but might as well keep it consistent.

Reply:
Lol good point, let me fix that :)